.github/workflows/build-container.yml

```
@@ -0,0 +1,35 @@
+name: build-docker-image
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '7 0 * * sun'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
+
+    steps:
+
+    - name: checkout docker-software-base
+      uses: actions/checkout@v2
+
+    - name: make date tag
+      id: mkdatetag
+      run: date +%Y%m%d-%H%M
```

```suggestion
      run: echo "::set-output name=timestamp::$(date +%Y%m%d-%H%M)"
```

AFAICT, you set variables by dumping `::set output name=<VAR NAME>::<VAL>` into stdout

---

Does that mean that we need `::set-output name=dtag::...` (instead of `name=timestamp::...`), since i refer to the output as `dtag`?

---

```suggestion
      run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
```

Ah yes, copy paste fail on my part: